integrations/snyk/snyk/client.py

```diff
@@ -270,36 +285,68 @@ async def _get_user_details(self, user_id: str | None) -> dict[str, Any]:
             else:
                 raise
 
+    async def _get_user_details(self, user_reference: str | None) -> dict[str, Any]:
+        if (
+            not user_reference
+        ):  ## Some projects may not have been assigned to any owner yet. In this instance, we can return an empty dict
+            return {}
+        user_id = user_reference.split("/")[-1]
+        user_reference = user_reference.replace("/rest", "")
+        cached_details = event.attributes.get(f"{CacheKeys.USER}-{user_id}")
```

Comment:
SInce `f"{CacheKeys.USER}-{user_id}"` is used a couple of times in this function, it's worth extracting to a variable in the top, to prevent accidental typos that can be DRY'd out.

Reply:
This has been fixed